common/lib/xmodule/xmodule/tests/test_video.py

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Why the blank line here? This may interfere with the UTF-8 file encoding declaration.